### DISCUSSION TITLE
Red and Blue colors are swapped. swapBytes(true/false) does not fix the issue.

### DISCUSSION THREAD

**Discussion by tavdog** [Bodmer/TFT_eSPI#2320] [Q&A - General]
Using T-Display-S3.   Mostly everything is working but red and blue are always swapped.  setting swapBytes(false) just makes all the colors wrong.  I'm banging my head against the wall trying to fix this.  Anybody seen anything like this ?
<img width="863" alt="Screen Shot 2023-01-13 at 1 22 51 PM" src="https://user-images.githubusercontent.com/8324295/212438006-0278db82-1274-4ab7-8d3d-e4288a3dca29.png">

**Comment by Bodmer** [comment]
You can define the colour order in the setup as RGB or BGR,  see here:
https://github.com/Bodmer/TFT_eSPI/blob/master/User_Setups/Setup206_LilyGo_T_Display_S3.h#L10
